### PR TITLE
Bog App - Controllers Specs

### PR DESCRIPTION
**Pull request by raminka13**
Ruby Rails
- Created a Request spec file for users and posts controllers.
- Each test action checks:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,11 +1,5 @@
...
-    @posts = Post.where(author_id: @user.id).order(created_at: :desc)
-  end
+  def index; end
 
-  def show
-    @post = Post.includes(:comments, comments: [:author]).find(params[:id])
-    @comments = Comment.where(post_id: @post.id).order(created_at: :desc)
-  end
+  def show; end
 end
```
Comment (ShoiraTa) [inline app/controllers/posts_controller.rb:5]: - [ ] The idea of adding specs in controllers is to make sure that controllers in app work as expected. I don't know why methods were deleted from controllers, please do not delete them, if some specs are failing, try to resolve the issues

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @raminka13 

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username [@shoirata].


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (raminka13) [reply app/controllers/posts_controller.rb:5]: @ShoiraTa I understand your expectation but per Microverse requirements from the previous milestone in the project [Blog app - controllers](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md), and this one, I'm doing everything necessary to get this PR approved. I don't need logic inside the controllers methods or the views. Please check that. I coded that logic just for fun.
If you want to point me in the right direction on how testing should be done including how to pass params in testing or how to populate the testing database, I would be very thankful.
Please approve this PR, as I meet all requirements.
Thanks.

**Review by AshakaE — COMMENTED** [review]
Hi @raminka13 , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**